### PR TITLE
Update 10_vis__dashboard_view.md

### PR DESCRIPTION
**Pull request by andriichumak**
Fixed a broken link

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- gooddata-ui-documentation-validation : SKIPPED
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/547/ : Tests on opensource repositories should be triggered by users in `gooddata` organization in 9s

File: docs/10_vis__dashboard_view.md [hunk 1]
```diff
@@ -257,7 +257,7 @@ clearDashboardViewCaches();
...
 | drillableItems               | false     | [IDrillableItem[]](15_props__drillable_item.md)  | An array of points and attribute values to be drillable                                                                                           |
 | onDrill                      | false     | Function                                         | A callback when a drill is triggered on any widget in the dashboard                                                                       |
-| theme                        | false     | ITheme                                           | The theme to be used for the dashboard (see [Theming](#theming))                                                                                      |
+| theme                        | false     | ITheme                                           | The theme to be used for the dashboard (see [Themes](#themes))                                                                                      |
```
Comment (plavemar) [inline docs/10_vis__dashboard_view.md:260]: Please update the same file on the 8.2.0 version as well.

Comment (andriichumak) [reply docs/10_vis__dashboard_view.md:260]: `website/versioned_docs/version-8.2.0/10_vis__dashboard_view.md` updated

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/839/ : SUCCESS in 58s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/552/ : SUCCESS in 6s

**Review by plavemar — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR448

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/841/ : SUCCESS in 58s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/149/ : SUCCESS in 40s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/47/ : SUCCESS in 35s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/237/ : SUCCESS in 16s

**Pull request merged**